### DISCUSSION TITLE
Avoid requery Lazy Load when onRowselect

### DISCUSSION THREAD

**Discussion by trinidadmarcelo** [primefaces/community#708] [PrimeFaces]
Hi, there is a way to avoid load method requery on lazy mode, when onRowselect or clic row?

@cagataycivici 

Thanks in advance!

**Comment by melloware** [comment]
Please provide an executable example using the [PrimeFaces Test](https://github.com/primefaces/primefaces-test) project.  It is the only way developers can debug your problem to help.
